webapp/src/templates/partials/contacts_list.html

```diff
@@ -1,9 +0,0 @@
-<div id="contacts-list" class="col-sm-4 inbox-items left-pane">
-  <ul class="unfiltered" ng-show="(!loading || appending) && !filtered"></ul>
-  <ul class="filtered" ng-show="(!loading || appending) && filtered"></ul>
-  <p ng-show="error" class="alert alert-danger" role="alert" translate>Error fetching contacts</p>
-  <p class="loading-status" ng-show="!error && !loading && !hasContacts" translate>No contacts found</p>
-  <p class="loading-status" ng-show="!error && !loading && hasContacts && !moreItems" translate>No more contacts</p>
-  <div class="loader" ng-show="loading"></div>
-  <div class="padding"></div>
-</div>
```

Comment:
Can you do a `git mv` rather than delete this file so we can save the history?

Reply:
Well I learned a lot about git in the process, and github can't seem to tell the difference, but the file histories are now preserved!